care/facility/api/serializers/prescription.py

```diff
@@ -127,6 +127,10 @@ def validate(self, attrs):
                 raise serializers.ValidationError(
                     {"indicator": "Indicator should be set for PRN prescriptions."}
                 )
+            if attrs.get("max_dosage") is None or attrs.get("max_dosage") < attrs.get("base_dosage"):
```

Comment:
```suggestion
            if attrs.get("max_dosage") and attrs.get("max_dosage") < attrs.get("base_dosage"):
```

base_dosage is already validated on line 120, and max_dosage is non mandatory

Reply:
I have done the changes as suggested above.